### ISSUE TITLE
LocalStack Stepfunction seems to add additional characters in the output 

### ISSUE THREAD

**Issue by 3mard** [localstack/localstack#1375]
I think this is a bug in upstrame aws local-sfn:

actuall aws response :
```
aws --profile om stepfunctions get-execution-history --execution-arn arn:aws:states:eu-west-1:967344299613:execution:my-state-machine:asd
{
    "events": [
        {
            "timestamp": 1560606024.632,
            "executionStartedEventDetails": {
                "input": "{\n   \"n1\":10,\"n2\":20\n}",
                "roleArn": "arn:aws:iam::967344299613:role/iam_for_sfn"
            },
            "type": "ExecutionStarted",
            "id": 1,
            "previousEventId": 0
        },
        {
            "timestamp": 1560606024.654,
            "type": "TaskStateEntered",
            "id": 2,
            "stateEnteredEventDetails": {
                "input": "{\n   \"n1\":10,\"n2\":20\n}",
                "name": "AddNumbers"
            },
            "previousEventId": 0
        },
        {
            "timestamp": 1560606024.654,
            "lambdaFunctionScheduledEventDetails": {
                "input": "{\n   \"n1\":10,\"n2\":20\n}",
                "resource": "arn:aws:lambda:eu-west-1:967344299613:function:add"
            },
            "type": "LambdaFunctionScheduled",
            "id": 3,
            "previousEventId": 2
        },
        {
            "timestamp": 1560606024.703,
            "type": "LambdaFunctionStarted",
            "id": 4,
            "previousEventId": 3
        },
        {
            "lambdaFunctionSucceededEventDetails": {
                "output": "{\"n1\":30}"
            },
            "timestamp": 1560606025.036,
            "type": "LambdaFunctionSucceeded",
            "id": 5,
            "previousEventId": 4
        },
        {
            "timestamp": 1560606025.036,
            "stateExitedEventDetails": {
                "output": "{\"n1\":30}",
                "name": "AddNumbers"
            },
            "type": "TaskStateExited",
            "id": 6,
            "previousEventId": 5
        },
        {
            "timestamp": 1560606025.043,
            "type": "TaskStateEntered",
            "id": 7,
            "stateEnteredEventDetails": {
                "input": "{\"n1\":30}",
                "name": "SquareNumber"
            },
            "previousEventId": 6
        },
        {
            "timestamp": 1560606025.043,
            "lambdaFunctionScheduledEventDetails": {
                "input": "{\"n1\":30}",
                "resource": "arn:aws:lambda:eu-west-1:967344299613:function:square"
            },
            "type": "LambdaFunctionScheduled",
            "id": 8,
            "previousEventId": 7
        },
        {
            "timestamp": 1560606025.052,
            "type": "LambdaFunctionStarted",
            "id": 9,
            "previousEventId": 8
        },
        {
            "lambdaFunctionSucceededEventDetails": {
                "output": "900"
            },
            "timestamp": 1560606025.405,
            "type": "LambdaFunctionSucceeded",
            "id": 10,
            "previousEventId": 9
        },
        {
            "timestamp": 1560606025.405,
            "stateExitedEventDetails": {
                "output": "900",
                "name": "SquareNumber"
            },
            "type": "TaskStateExited",
            "id": 11,
            "previousEventId": 10
        },
        {
            "executionSucceededEventDetails": {
                "output": "900"
            },
            "timestamp": 1560606025.405,
            "type": "ExecutionSucceeded",
            "id": 12,
            "previousEventId": 11
        }
    ]
}
```

local stack response :
```
awslocal stepfunctions get-execution-history --execution-arn arn:aws:states:us-east-1:000000000000:execution:my-state-machine:c945d814bccb-4dd4-80ac-601250aa2fef
{
    "events": [
        {
            "timestamp": 1560606653.504, 
            "executionStartedEventDetails": {
                "input": "{\"n1\":10,\"n2\":20}", 
                "roleArn": "arn:aws:iam::000000000000:role/iam_for_sfn"
            }, 
            "type": "ExecutionStarted", 
            "id": 1, 
            "previousEventId": 0
        }, 
        {
            "timestamp": 1560606653.504, 
            "type": "TaskStateEntered", 
            "id": 2, 
            "stateEnteredEventDetails": {
                "input": "{\"n1\":10,\"n2\":20}", 
                "name": "AddNumbers"
            }, 
            "previousEventId": 0
        }, 
        {
            "timestamp": 1560606653.504, 
            "lambdaFunctionScheduledEventDetails": {
                "input": "{\"n1\":10,\"n2\":20}", 
                "resource": "arn:aws:lambda:us-east-1:000000000000:function:add"
            }, 
            "type": "LambdaFunctionScheduled", 
            "id": 3, 
            "previousEventId": 2
        }, 
        {
            "timestamp": 1560606653.504, 
            "type": "LambdaFunctionStarted", 
            "id": 4, 
            "previousEventId": 3
        }, 
        {
            "lambdaFunctionSucceededEventDetails": {
                "output": "{\"n1\":30}\n"
            }, 
            "timestamp": 1560606654.94, 
            "type": "LambdaFunctionSucceeded", 
            "id": 5, 
            "previousEventId": 4
        }, 
        {
            "timestamp": 1560606654.94, 
            "stateExitedEventDetails": {
                "output": "{\"n1\":30}\n", 
                "name": "AddNumbers"
            }, 
            "type": "TaskStateExited", 
            "id": 6, 
            "previousEventId": 5
        }, 
        {
            "timestamp": 1560606654.94, 
            "type": "TaskStateEntered", 
            "id": 7, 
            "stateEnteredEventDetails": {
                "input": "{\"n1\":30}\n", 
                "name": "SquareNumber"
            }, 
            "previousEventId": 0
        }, 
        {
            "timestamp": 1560606654.94, 
            "lambdaFunctionScheduledEventDetails": {
                "input": "{\"n1\":30}\n", 
                "resource": "arn:aws:lambda:us-east-1:000000000000:function:square"
            }, 
            "type": "LambdaFunctionScheduled", 
            "id": 8, 
            "previousEventId": 7
        }, 
        {
            "timestamp": 1560606654.94, 
            "type": "LambdaFunctionStarted", 
            "id": 9, 
            "previousEventId": 8
        }, 
        {
            "lambdaFunctionSucceededEventDetails": {
                "output": "\"900\"\n"
            }, 
            "timestamp": 1560606656.605, 
            "type": "LambdaFunctionSucceeded", 
            "id": 10, 
            "previousEventId": 9
        }, 
        {
            "timestamp": 1560606656.605, 
            "stateExitedEventDetails": {
                "output": "\"900\"\n", 
                "name": "SquareNumber"
            }, 
            "type": "TaskStateExited", 
            "id": 11, 
            "previousEventId": 10
        }, 
        {
            "executionSucceededEventDetails": {
                "output": "\"900\"\n"
            }, 
            "timestamp": 1560606656.605, 
            "type": "ExecutionSucceeded", 
            "id": 12, 
            "previousEventId": 0
        }
    ]
}
```

`"output": "900"` vs `"output": "\"900\"\n"`

**Comment by whummer** [comment]
Thanks for reporting @3mard . This could be related to the way we're returning results from Lambda invocations. This has potentially been fixed in #1885.

Before we dig deeper into this issue, can you please give it another try with the `latest` version of the Docker image? Thanks!

**Comment by whummer** [comment]
Closing this issue due to inactivity. Please report here or create a follow-up issue with detailed steps to reproduce if the problem persists. Thanks!
